$NetBSD: patch-ac,v 1.10 2009/05/17 23:11:21 dholland Exp $

Hack around misuse of math.h.
Upstream: already fixed upstream in coreutils 7.x.

--- src/seq.c.orig  Mon May  4 16:07:31 2009
+++ src/seq.c       Mon May  4 16:08:05 2009
@@ -20,7 +20,6 @@
 #include <getopt.h>
 #include <stdio.h>
 #include <sys/types.h>
-#include <math.h>
 #include <float.h>

 #include "system.h"
